migrations/v3.js

```diff
@@ -0,0 +1,322 @@
+import { checkContent, describe, getConfig, getCourse, mutateContent, testStopWhere, testSuccessWhere, whereContent } from 'adapt-migrations';
+import _ from 'lodash';
+
+describe('core - update to v3.0.0', async () => {
+  let articles, blocks, components, config, contentObjects, course;
+  const newArticleAttributes = {
+    instruction: '',
+    _isHidden: false,
+    _isVisible: true,
+    _ariaLevel: 0,
+    _onScreen: {
+      _isEnabled: false,
+      _classes: '',
+      _percentInviewVertical: 50
+    }
+  };
+  const newBlockAttributes = {
+    instruction: '',
+    _isHidden: false,
+    _isVisible: true,
+    _ariaLevel: 0,
+    _onScreen: {
+      _isEnabled: false,
+      _classes: '',
+      _percentInviewVertical: 50
+    }
+  };
+  const newComponentAttributes = {
+    _isHidden: false,
+    _isVisible: true,
+    _ariaLevel: 0,
+    _disableAccessibilityState: false,
+    _onScreen: {
+      _isEnabled: false,
+      _classes: '',
+      _percentInviewVertical: 50
+    }
+  };
+  const newContentObjectAttributes = {
+    instruction: '',
+    _isHiddenFromMenu: false,
+    _isHidden: false,
+    _isVisible: true,
+    _ariaLevel: 0,
+    _disableAccessibilityState: false,
+    _onScreen: {
+      _isEnabled: false,
+      _classes: '',
+      _percentInviewVertical: 50
+    }
+  };
+  const completionCriteria = {
+    _requireContentCompleted: true,
+    _requireAssessmentCompleted: false
+  };
+  const redundantConfigAccessibility = [
+    '_isEnabledOnTouchDevices',
+    '_shouldSupportLegacyBrowsers'
+  ];
+  const newAriaLevels = {
+    _menu: 1,
+    _menuItem: 2,
+    _page: 1,
+    _article: 2,
+    _block: 3,
+    _component: 4,
+    _componentItem: 5
+  };
+  const redundantAriaLabels = [
+    'menu',
+    'menuItem',
+    'menuViewButton',
+    'navigationBack',
+    'closeResources',
+    'drawerBack',
+    'menuBack'
+  ];
+
+  whereContent('core - where contentObjects', async (content) => {
```

Comment:
```suggestion
  whereContent('core - where contentObjects', async (content) => {
```

This code is "where articles"